crates/emmylua_ls/src/handlers/test_lib/mod.rs

```diff
@@ -195,29 +191,24 @@ impl ProviderVirtualWorkspace {
             CancellationToken::new(),
         );
         let Some(result) = result else {
-            return false;
+            panic!("completion result is empty");
```

Comment:
If panic occurs inside the function, does that mean I can’t quickly jump to the exact error location when an error happens?